src/scripts/components/MultiAvatar.jsx

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class MultiAvatar extends React.Component {
+  renderInitials(initials) {
+    return (
+      <text x="50" y="50" textAnchor="middle" dy="0.35em" fontSize="30">{initials}</text>
+    );
+  }
+
+  renderAvatars() {
+    const { images, names } = this.props;
+    const firstUserIntitials = names[0];
```

Comment:
Is this expecting initials or a name? In the original avatar component we pass the full name and then render initials off of that. Would be good to maintain consistency.

Reply:
Soooo, yes and no?

https://github.com/rhinogram/rhinofront/blob/ab695de2b44f9b1c339dd9d575ba1bbe6aa3c461/src/components/ChatInbox.jsx#L48

If you look at ☝️ , and check where we render <Avatar /> in rhinofront, you'll see that in almost all cases we're passing in the initials. Seems that the logic in Avatar.jsx is enforcing that here(basically, accepts either initials or a name):
```
if (name) {
      splitName = name.split(' ');
      initials = splitName[0][0] + splitName[splitName.length - 1][0];
    }
```

I'll add the same check here so it can _expect_ a name, but also accept initials. _That_ would be consistent.

Reply:
done